hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -50,6 +50,19 @@ profile::jenkinscontroller::jcasc:
               discarder:
                 logRotator:
                   numToKeepStr: "5"
+  artifact-manager:
+    data: |-
+      aws:
+        awsCredentials:
+          credentialsId: "aws-s3-artifact-manager"
+          region: "us-east-2"
+        s3:
+          container: "ci-jenkins-io-artifacts"  # Resource defined in jenkins-infra/aws/ci.jenkins.io.tf
+          disableSessionToken: false            # For non AWS services only
```

Comment:
Note that some of these are set to the default value and _could_ be omitted.

Reply:
We've got bitten by sudden changes on defaults values in JCasc (for instance `ec2` plugin) so we prefer being explicit 😅